.nvmrc

```diff
@@ -1 +1 @@
-18.0.0
+24.11.1
```

Comment:
Yarn v4 is not available in Node 18, so install the [current LTS version](https://nodejs.org/en/download/archive/v24.11.1).